pkg/kubelet/watchdog/watchdog_linux_test.go

```diff
@@ -84,7 +83,7 @@ func TestNewHealthChecker(t *testing.T) {
 				enabledErr: tt.mockErr,
 			}
 
-			_, err := NewHealthChecker(&mockSyncLoopHealthChecker{}, WithWatchdogClient(mockClient))
+			_, err := NewHealthChecker(klog.FromContext(ktesting.Init(t)), &mockSyncLoopHealthChecker{}, WithWatchdogClient(mockClient))
```

Comment:
you can use [ktesting.NewTestContext](https://github.com/kubernetes/kubernetes/blob/master/vendor/k8s.io/klog/v2/ktesting/setup.go#L33) to get a context **and** a logger.